### ISSUE TITLE
*ABVersion directive not enabled

### ISSUE THREAD

**Issue by 127001** [protoarch/angular#3]
I tried to add the npm module following your instructions, but after adding

`<ng-container *ABVersion="'old'">
  Old version goes here
</ng-container>

<ng-container *ABVersion="'new'">
  New version goes here
</ng-container>`

I end up with the following error in my console: 

`NG0303: Can't bind to 'ABVersion' since it isn't a known property of 'ng-container'.`

Not sure if I missed something.
I first tried to add the module to an existing project, but this also happens on a demo project I just created: https://github.com/127001/angular-test-project

**Comment by Romanchuk** [comment]
@127001 Sorry, docs a bit old 
its
`*prtAbVersion=`
now

**Comment by Romanchuk** [comment]
I should update docs since it migrated from other reps

**Comment by iznogood112** [comment]
i saw this as well after checking your directive, but this still doesnt work and gives the same error.

```
<pre>hello world</pre>
<ng-container *prtAbVersion="'old'">
  Old version goes here
</ng-container>

<ng-container *prtAbVersion="'new'">
  New version goes here
</ng-container>
```

`NG0303: Can't bind to 'prtAbVersion' since it isn't a known property of 'ng-container'.`


app.module.ts:

```
import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';
import { AbModule } from '@protoarch.angular/ab';

import { AppComponent } from './app.component';

@NgModule({
  declarations: [
    AppComponent
  ],
  imports: [
    AbModule.forRoot([
      {
        versions: [ 'old', 'new' ],
        versionForCrawlers: 'old',
        expiration: 45,
      },
    ]),
    BrowserModule
  ],
  exports: [AbModule],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }

```

**Comment by 127001** [comment]
I've tried your suggestion, but it still gives the same error. It now has a second error as well, versions is not iterable:

![Screenshot from 2021-12-28 20-19-36](https://user-images.githubusercontent.com/149222/147599897-72e87df0-4203-4afa-a9e2-f3cd84cd5991.png)

I've updated my demo code with your suggestion at: https://github.com/127001/angular-test-project

**Comment by Romanchuk** [comment]
@127001 @iznogood112 Thx guys, fixed in 6.0.1
Also added module to demo app (in this repo)

**Comment by 127001** [comment]
Thanks! And I can confirm that on my test project.
